pkg/operator/ceph/config/conditions.go

```diff
@@ -118,7 +118,7 @@ func translatePhasetoState(phase cephv1.ConditionType) cephv1.ClusterState {
 }
 
 // Updating the status of Progressing, Updating or Upgrading to False once cluster is Ready
-func checkConditionFalse(context *clusterd.Context, namespace, name string) {
+func CheckConditionFalse(context *clusterd.Context, namespace, name string) {
 	tempConditionList := []cephv1.ConditionType{cephv1.ConditionUpdating, cephv1.ConditionUpgrading, cephv1.ConditionProgressing}
 	var tempCondition cephv1.ConditionType
 	for _, conditionType := range tempConditionList {
```

Comment:
I'm confused by this method. Is the intent to update all three of these conditions to false? Looks like it only ever updates the last one in the list and will ignore the others.

Reply:
Fixed this.